Makefile

```diff
@@ -41,6 +41,7 @@ bootstrap: toolset
 	mkdir -p $(BOOTSTRAP_PATH) ; \
 	cp Binaries/$(BUILD_CONFIGURATION)/csccore/* $(BOOTSTRAP_PATH) ; \
 	cp Binaries/$(BUILD_CONFIGURATION)/vbccore/* $(BOOTSTRAP_PATH) ; \
+	./build/scripts/crossgen.sh $(BOOTSTRAP_PATH) ; \
```

Comment:
If they're identical let's pick one way of doing it and standardize on that.  I'm ambilevent as to which way.